frontend/src/modules/editor/contentObject/views/editorPageArticleView.js

```diff
@@ -11,25 +11,37 @@ define(function(require){
     tagName: 'div',
 
     events: _.extend({}, EditorOriginView.prototype.events, {
-      'click a.add-block': 'addBlock',
-      'click a.article-delete': 'deleteArticlePrompt',
-      'click a.open-context-article': 'openContextMenu',
-      'dblclick': 'loadArticleEdit'
+      'click .add-block': 'addBlock',
+      'click .article-delete': 'deleteArticlePrompt',
+      'click .open-context-article': 'openContextMenu',
+      'dblclick .editor-page-element-detail-inner,.article-blocks': 'loadArticleEdit',
```

Comment:
Please restore the original generic `dblclick` listeners so we may click anywhere to edit e.g. on an article's body text.

The code you've implemented in `editorOriginView.js` should be enough to prevent double clicking on buttons?